pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: form_builder_validators
 description: Form Builder Validators set of validators for FlutterFormBuilder. Provides common validators and a way to make your own.
-version: 8.2.0
+version: 8.2.1
```

Comment:
Don't need this. We update version when start a release process.

Reply:
OK, thanks for hint.